Wikipedia/UI-V5/WMFNearbyViewModel.m

```diff
@@ -0,0 +1,193 @@
+//
+//  WMFNearbyViewModel.m
+//  Wikipedia
+//
+//  Created by Brian Gerstle on 9/9/15.
+//  Copyright (c) 2015 Wikimedia Foundation. All rights reserved.
+//
+
+#import "WMFNearbyViewModel.h"
+
+// Location & Fetching
+#import "WMFLocationManager.h"
+#import "WMFLocationSearchFetcher.h"
+
+// Models
+#import "WMFLocationSearchResults.h"
+#import "MWKLocationSearchResult.h"
+#import "MWKHistoryEntry.h"
+#import "CLLocation+WMFApproximateEquality.h"
+#import "CLLocation+WMFBearing.h"
+
+// Frameworks
+#import "Wikipedia-Swift.h"
+#import "PromiseKit.h"
+
+@interface WMFNearbyViewModel ()
+<WMFNearbyControllerDelegate>
+
+@property (nonatomic, strong) WMFLocationSearchFetcher* locationSearchFetcher;
+@property (nonatomic, strong) WMFLocationManager* locationManager;
+@property (nonatomic, assign) NSUInteger resultLimit;
+
+@property (nonatomic, strong, readwrite) WMFLocationSearchResults* locationSearchResults;
+
+@property (nonatomic, strong) NSMutableIndexSet* autoUpdatingIndexes;
```

Comment:
What determines if a result is "auto-updated" or not. Does this mean some results don't get updates? 

Also `autoUpdatingIndexes ` seems very generic/dubious. Really only the bearing and distance are getting updated right? Or is there something else?